### DISCUSSION TITLE
How do I adjust the spaces in between the pinned applications in the taskbar?

### DISCUSSION THREAD

**Discussion by Archilyte** [valinet/ExplorerPatcher#443] [Q&A]
For some reason the spaces between then is a little bigger, causing the whole taskbar to look "streetched out". How do I change this?

**Comment by Anixx** [comment]
Use quick launch toolbar

**Comment by chip33** [comment]
[7+ Taskbar Tweaker](https://rammichael.com/7-taskbar-tweaker) has an option to reduce gaps between pinned items on the taskbar.
